eoscostarica/proton-affiliate
# feat(webapp): implement add user

### Title

### What does this PR do?

- Resolve #86

### Steps to test

1. make run
2. login with admin user
3. click on "+" button
4. type an account
5. click on "add" button 
6. confirm the message 

#### CheckList
